docs/content/en/docs/fleet-manager/policy.md

```diff
@@ -34,6 +41,16 @@ After a while, we can see the fleet is `ready`:
 kubectl wait fleet quickstart --for='jsonpath='{.status.phase}'=Ready'
 ```
 
+Thanos and Grafana are installed correctly:
```

Comment:
? Shouldnot this be in the above doc, here it is installing policy engine

Reply:
my mistake, fixed.